### ISSUE TITLE
Delegaters list

### ISSUE THREAD

**Issue by wirednkod** [delegit-xyz/dashboard#187]
We should add some mention about the list and where someone can add the details of the delegator (Novasama jsons).

**Comment by Tbaut** [comment]
Agreed. What do you think would be a good place? I was thinking of a menu entry "Becode delegate". Other than that I've a hard time knowing how to promote this in a nice way. Another idea, now that we have some free space at the top (bc we'll remove 1 card) we could have a dismissible card, with a nice design, where users could close it and never see it again (we store the close action in localstorage) where we redirect them to a page in the wiki of this repo, with a little explanation?

**Comment by Tbaut** [comment]
Hey @dbpatty the list is actually fetched automatically from the nova repo, make a PR there, and it'll be taken from there: https://github.com/novasamatech/opengov-delegate-registry/tree/master/registry

**Comment by dbpatty** [comment]
Ty for your quick reply
